src/driver/Firestore/InProcessFirestore.ts

```diff
@@ -112,7 +112,25 @@ export class InProcessFirestore implements IFirestore {
     }
 
     collectionGroup(collectionId: string): IFirestoreQuery {
-        throw new Error("Not implemented")
+        const storage = this._getPath([""]) || {}
+        const newFirestore = new InProcessFirestore()
+
+        const flattenStorage: any = (collection: any, key: string) => {
```

Comment:
is `collection` the best name for the parameter? Could it be `storage` instead?

Reply:
storage name is already been used so we can't use it again, I wanted to make the names within this little method to be more generic. But we can rename it something else.
@nikbolotin